README.md

```diff
@@ -74,10 +74,7 @@ page](https://docs.xarray.dev/en/latest/contributing.html#).
 - Ask usage questions ("How do I?") on
     [GitHub Discussions](https://github.com/pydata/xarray/discussions).
 - Report bugs, suggest features or view the source code [on
-    GitHub](https://github.com/pydata/xarray).
-- For less well defined questions or ideas, or to announce other
-    projects of interest to xarray users, use the [mailing
-    list](https://groups.google.com/forum/#!forum/xarray).
+  GitHub](https://github.com/pydata/xarray).
```

Comment:
We should add

`[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)`

To the badges.

(Or maybe link to the new internal license file instead of the Weblink)